frontend/src/backendClient.ts

```diff
@@ -38,8 +38,8 @@ const axiosInstance = axios.create({
   withCredentials: true,
 });
 
-export async function getArticles(): Promise<PaginationResponse<Article>> {
-  const response = await axiosInstance.get(`/api/articles`);
+export async function getArticles(query?: string): Promise<PaginationResponse<Article>> {
+  const response = await axiosInstance.get(`/api/articles${query ? `?search=${query}` : ""}`);
```

Comment:
Should we use a request configuration and params here?

See: https://axios-http.com/docs/req_config

```suggestion
 const params = {}
 if (query) params.search = query
  const response = await axiosInstance.get("/api/articles", {params});
```